__tests__/vending-machine.spec.js

```diff
@@ -0,0 +1,251 @@
+
+const VendingMachine = require('../src/vending-machine');
+
+describe('vendingMachine', () => {
+  let test = {};
+
+  beforeEach(() => {
+
+    test.inventory = {
+      1: {
+        productId: 1,
+        productName: 'mars bar',
+        productPrice: 1.75,
+        productQuantity: 10
+      },
+      2: {
+        productId: 2,
+        productName: 'twix',
+        productPrice: 2.00,
+        productQuantity: 10
+      },
+      3: {
+        productId: 3,
+        productName: 'm&ms',
+        productPrice: 2.50,
+        productQuantity: 10
+      },
+      4: {
+        productId: 4,
+        productName: 'payday',
+        productPrice: 3.15,
+        productQuantity: 10
+      }
+    }
+
+    test.vendingMachine = new VendingMachine(test.inventory);
+  });
+
+  describe('.restock', () => {
+
+    test.itemId = 0;
+    test.newItemsAdded = 10;
+
+    beforeEach(() => {
+      test.itemId++;
+      test.subject = new VendingMachine(test.inventory);
+      test.subject.restockItem({ productId: test.itemId, productQuantity: test.newItemsAdded });
+    });
+
+    it('should increment the first item\'s productQuantity by 10', () => {
+      expect(test.subject.inventory[1].productQuantity).toEqual(20)
+    });
+
+    it('should increment the second item\'s productQuantity by 10', () => {
+      expect(test.subject.inventory[2].productQuantity).toEqual(20)
+    });
+
+    it('should increment the third item\'s productQuantity by 10', () => {
+      expect(test.subject.inventory[3].productQuantity).toEqual(20)
+    });
+
+    it('should increment the fourth item\'s productQuantity by 10', () => {
+      expect(test.subject.inventory[4].productQuantity).toEqual(20)
```

Comment:
I would recommend getting data that is more varied. When every quantity equals 20, it's hard to know if you're testing the wrong thing.